src/Bindings/LuaFunctions.h

```diff
@@ -6,7 +6,7 @@
 
 inline unsigned int GetTime()
 {
-	return (unsigned int)time(0);
+	return static_cast<unsigned int>(time(0));
```

Comment:
This needs to be rewriten to std::chrono, because this cast is undefined in general.